doc/sphinx/user/methods/melt-transport.md

```diff
@@ -13,44 +14,24 @@ through and interacting with a viscously deforming host rock. This requires
 the advection of a compositional field representing the volume fraction of
 melt present at any given time (the porosity $\phi$), and also a change of the
 mechanical part of the system. The latter is implemented using the approach of
-(Keller, May, and Kaus 2013) and changes the Stokes system to
+{cite:t}`keller:etal:2013` and changes the Stokes system to
 ```{math}
+:label: eq:stokes-1-melt
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf{u}_s) - \frac{1}{3}(\nabla \cdot \mathbf{u}_s)\mathbf 1\right)\right] + \nabla p_f + \nabla p_c  = \rho \mathbf g   \qquad   \textrm{in $\Omega$}, \\
```

Comment:
Please keep the formatting of these equation and the ones below. Previously the individual terms were separated so that they were somewhat readable, the long lines make them very hard to read.